versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBStore.java

```diff
@@ -106,8 +105,19 @@ public void start() {
         final List<ColumnFamilyHandle> columnFamilyHandles = new ArrayList<>();
         transactionDB = OptimisticTransactionDB.open(dbOptions, dbPath, columnFamilies, columnFamilyHandles);
         rocksDB = transactionDB.getBaseDB();
-        valueTypeToColumnFamily = IntStream.rangeClosed(1, ValueType.values().size()).boxed().collect(
-          ImmutableMap.toImmutableMap(k -> ValueType.values().get(k - 1), columnFamilyHandles::get));
+        valueTypeToColumnFamily = ValueType.values().stream()
+          .collect(ImmutableMap.<ValueType<?>, ValueType<?>, ColumnFamilyHandle>toImmutableMap(
+            v -> v,
+            v -> {
+              String collectionName = v.getTableName(config.getTablePrefix());
+              // TODO: this is probably not correct. I guess we need to look up the columnFamilyHandle based on prefix
+              // or ColumnFamilyDescriptor.
+              return rocksDB.getDefaultColumnFamily();
```

Comment:
This is indeed incorrect, you need to map the column families returned in columnFamilyHandles to the names.

Reply:
Rewritten